tests/all/integration/api/group/test_group_misc.py

```diff
@@ -0,0 +1,140 @@
+import json
+
+from flask_jwt_extended import create_access_token
+
+from app.api.helpers.db import get_or_create
+from app.models.role import Role
+from app.models.users_events_role import UsersEventsRoles
+from tests.factories.event import EventFactoryBasic
+from tests.factories.group import GroupFactory
+from tests.factories.user import UserFactory
+
+
+def test_group_post_access_allow(db, client):
+    new_user = UserFactory(is_admin=False, is_verified=True)
+    event = EventFactoryBasic()
+    role, _ = get_or_create(Role, name='coorganizer', title_name='Co-organizer')
+    UsersEventsRoles(user=new_user, event=event, role=role)
+
+    db.session.commit()
+    data = json.dumps(
+        {
+            "data": {
+                "type": "group",
+                "relationships": {
+                    "user": {"data": {"id": str(new_user.id), "type": "user"}},
```

Comment:
User should be dumps only. I should not be able to make you the creator of the group